torch/csrc/jit/test_jit.cpp

```diff
@@ -929,6 +931,12 @@ TEST_CASE( "jit test CUDA", "[cuda]" ) {
 
 #endif
 
+
+void testProto() {
+  onnx_c2::ModelProto proto;
```

Comment:
Use `::ONNX_NAMESPACE` instead. In the CI we have set it to `ONNX_NAMESPACE_FOR_C2_CI` to explicitly check for people hardcoding "onnx_c2" :-)